### PR TITLE
Mirror backup

### PR DESCRIPTION
**Pull request by bobfuru**
This is a follow-up to address the docs added by @bgilbert in https://github.com/openshift/openshift-docs/pull/28637. With this PR, I've made small adjustments to match OCP docs style and guidelines.

### CODE REVIEW

**Comment by bobfuru** [comment]
Here is a preview build for review: https://github.com/openshift/openshift-docs/pull/29038

@bgilbert PTAL with a couple of notes: 

1) I had some issues with adding a commit in your original PR (due to user input error; user=me). I've hopefully included everything from there (https://github.com/openshift/openshift-docs/pull/28637) in this PR, with some updates based on our style and doc guidelines. Once this PR is merged, I'll close yours. 

Also, note that due to current Customer Portal docs limitations, we are unable to include xrefs to other modules. There are plans in future development to allow this behavior, but for now, we have to exclude these. You'll notice where I've made some wording replacements in some of the annotation notes to remove these.

@miabbott Also PTAL for any notable issues.

Thanks!

File: modules/installation-special-config-mirrored-disk.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+----
+variant: rhcos
+version: 0.1.0
+ignition:
+  config:
+    merge:
+      - local: worker.ign <1>
+boot_device:
+  mirror:
+<<<<<<< HEAD
```
Comment (bgilbert) [inline modules/installation-special-config-mirrored-disk.adoc:58]: Unresolved conflict marker.

**Comment by bgilbert** [comment]
For ease of review, I've applied your changes in a separate commit in the original PR (#28637) and will continue the review over there. :arrow_left:

File: modules/installation-special-config-mirrored-disk.adoc (hunk 1 above)
Comment (bobfuru) [reply modules/installation-special-config-mirrored-disk.adoc]: fixed for posterity, thanks

**Pull request closed without merging**